### PR TITLE
update read.me (remove unaccesible links)

### PR DESCRIPTION
**Pull request by jpyan**
remove links in the README to specific courses  for issue #810

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ You may also optionally place the json settings directly into the `ENV_JSON` env
     username: root
     password: root
     ```
-1. As you are now logged in as `root`, there are no courses listed. Navigate to http://localhost:5001/courses/231768 or http://localhost:5001/courses/430174 to view a sample course as a superuser.
+1. As you are now logged in as `root`, there are no courses listed.
```
Comment (justin0022) [inline README.md]: Could you add in a line about how to look up the courseid in the database and navigating to `http://localhost:5001/courses/{courseid}`?

Comment (jpyan) [reply README.md]: Absolutely. One sec

**Comment by jpyan** [comment]
Updated

File: README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ You may also optionally place the json settings directly into the `ENV_JSON` env
     username: root
     password: root
     ```
-1. As you are now logged in as `root`, there are no courses listed. Navigate to http://localhost:5001/courses/231768 or http://localhost:5001/courses/430174 to view a sample course as a superuser.
+1. As you are now logged in as `root`, there are no courses listed. Go through the latest SQL file you downloaded from the link in the setup and search for the line "INSERT INTO `course` VALUES". You can find the course_id in the second column of the tuples. And then nagivate to http://localhost:5001/courses/{course_id} with the course_id you found. (For example, with SQL file myla_test_data_2019_10_16.sql, on line 323, you can find course_id = 235420 or course_id = 362855. And then nagivate to http://localhost:5001/courses/235420 or http://localhost:5001/courses/362855 and you can view the course page as an admin.)
```
Comment (justin0022) [inline README.md]: I'd recommend users to connect to the SQL database directly and looking at the `course` table, `canvas_id` column.

**Comment by jpyan** [comment]
Ahh ur right! I will update it later

**Comment by jpyan** [comment]
You are welcome!

**Pull request merged**